[Doc Improvement][Revamp meeting extensibility public docs][2557380] (#6722)
MicrosoftDocs/msteams-docs: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

# Apps for Teams meetings and calls

> Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:

Reviewer: Suggestion, there might be some confusion on the Teams meeting app itself and separate 3P apps that interact with the meeting
```suggestion
Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting space can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:
```

:::image type="content" source="../assets/images/apps-in-meetings/meetingappextensibility.png" alt-text="The screenshot shows you how meeting app extensibility works.":::

An application developer must be familiar with the following product concepts to create custom meeting experiences in Microsoft Teams with apps.

> 

Reviewer 2: After this intro, I would recommend a short summary of the different capabilities of meeting apps (tabs, stage, bot, etc.). This page is a chance to really sell our meeting features and make them seem exciting!

Reviewer 3: differing for next iteration https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/2694775

# Apps for Teams meetings and calls

Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:

:::image type="content" source="../assets/images/apps-in-meetings/meetingappextensibility.png" alt-text="The screenshot shows you how meeting app extensibility works.":::

An application developer must be familiar with the following product concepts to create custom meeting experiences in Microsoft Teams with apps.

## Meeting lifecycle

Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.

> ### Types of Teams meetings

Author: updated the doc

> # Apps for Teams meetings and calls

Reviewer 3: @v-sdhakshina  I don't see a reference of Meeting apps capabilities in https://support.microsoft.com/en-us/office/meetings-1f8862a1-48f2-4869-863d-564123343d38#ID0EBBD=Participate; Should we add one or more sections?! 
cc: Manasi Sharma

Reviewer 3: do it as separate task

* [**Group calls**](https://support.microsoft.com/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in group chat.
* [**Instant meetings**](https://support.microsoft.com/office/start-an-instant-meeting-in-teams-ff95e53f-8231-4739-87fa-00b9723f4ef5): Meetings initiated through **Meet now** button in Teams calendar.
* [**Webinar**](https://support.microsoft.com/office/get-started-with-teams-webinars-42f3f874-22dc-4289-b53f-bbc1a69013e3): Webinar initiated through **Webinar** button under **New Meeting** dropdown.

> [!NOTE]
>
> * Apps for instant meetings, scheduled public channel meetings, one-on-one, and group calls are currently available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
> * Currently app experiences are not supported in Teams instant channel meetings.

Learn more about [Teams meetings, expiration and policies](/MicrosoftTeams/meeting-expiration) and [Meetings, webinars, and live events](/microsoftteams/quick-start-meetings-live-events).

## Meeting lifecycle

> Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.

Reviewer: Is attendee's **status** the same as user type here (federated, guest, etc.) or is it related to attendee vs presenter vs organizer?

If dependent on meeting role:
```suggestion
The meeting lifecycle includes pre-meeting, in-meeting, and post-meeting scenarios. These experiences may vary depending on a user's role in the meeting.
```

If dependent on user type:
```suggestion
The meeting lifecycle includes pre-meeting, in-meeting, and post-meeting scenarios. These experiences may vary depending on the type of user.
```

Suggest to link to user types and/or meeting roles above on "**role in the meeting**" or "**type of user**". There could be confusion here on the connection between attendee status and user type.

## User roles in Teams meeting

> The default participant settings are determined by an organization's IT administrator. The following are the participant roles in a meeting:

Reviewer 4: > The default participant settings are determined by an organization's IT administrator.

Let's move this to a separate paragraph, it's out of place in this one, especially as the topic sentence.

> * **Organizer**: The organizer schedules a meeting, sets the meeting options, assigns meeting roles, and starts the meeting. The users with Microsoft 365 account and Teams license can only be the organizers, and control attendee permissions. A meeting organizer can change the settings for a specific meeting. Organizers can make these changes on the **Meeting options** web page.

Reviewer: ```suggestion
* **Organizer**: The organizer schedules a meeting, sets the meeting options, assigns meeting roles, and starts the meeting. Only users with a Microsoft 365 account and Teams license can be the organizers, and control attendee permissions. A meeting organizer can change the settings for a specific meeting. Organizers can make these changes on the **Meeting options** web page.
```

Suggest to also include screenshot and/or link of Meeting options web page for clarity

> * **Presenter**: The presenters have same capabilities of the organizers with exclusions. A presenter can't remove an organizer from the session or modify meeting options for the session.

Reviewer: ```suggestion
* **Presenter**: Presenters in a meeting have similar capabilities as the organizer, with the exception of removing an organizer from the session and modifying meeting options for the session.
```

* **Attendee**: An attendee is a user who is invited to attend the meeting. Attendees have limited capabilities during the meeting.

> [!NOTE]
> Only an organizer or presenter can add, remove, or uninstall apps.

For more information, see [roles in a Teams meeting](https://support.microsoft.com/office/roles-in-a-teams-meeting-c16fa7d0-1666-4dde-8686-0a0bfe16e019).

> After you design your app based on participant roles in a meeting, you can identify each user type for meetings and select what they can access.

Reviewer 4: What does "user type" refer to here? Role? Let's use terminology consistently.

> After you design your app based on participant roles in a meeting, you can identify each user type for meetings and select what they can access.

Reviewer 4: > you can identify each user type for meetings

Can we link to how?

> [!TIP]
>
> * The default participant settings are determined by an organization's IT administrator. As per default settings, participants joining a meeting have the presenter role.

> > * Participant role isn't available in one-on-one call.

Reviewer 4: Did you mean presenter? Participant isn't one of the roles above.

Reviewer 3: @v-sdhakshina pls update "Presenter" instead of "Participant"

* **Federated or external**: A federated user is an external Teams user in another organization who has been invited to join a meeting. Federated users have valid credentials with federated partners and are authorized by Teams. They don't have access to your teams or other shared resources from your organization. Guest access is a better option for external users to have access to teams and channels. For more information, see [manage external access in Teams](/microsoftteams/manage-external-access).

    > [!NOTE]
    > Your Teams users can add apps when they host meetings or chats with other organizations. The users can use apps shared by external users when your users join meetings or chats hosted by other organizations. The data policies of the hosting user's organization, as well as the data sharing practices of the third-party apps shared by that user's organization, will be in effect.

    > [!IMPORTANT]

>     > Currently, third-party apps are available in Government Community Cloud (GCC) but are not available for GCC-High and Department of Defense (DOD). Third-party apps are turned off by default for GCC. To turn on third-party apps for GCC, see [manage app permission policies](/microsoftteams/teams-app-permission-policies) and [manage apps](/microsoftteams/manage-apps).

Reviewer 2: This sort of information (gov cloud) may be better in a Meeting FAQ page or closer to the bottom of the page, as it will only be relevant to a small minority of folks reading this page.

Reviewer 3: @v-sdhakshina  pls mark this resolved

    > [!NOTE]
    > Anonymous users inherit the global default user-level app permission policy. For more information, see [manage Apps](/microsoftteams/non-standard-users#anonymous-user-in-meetings-access).

## User roles in Teams meeting

The following are the user roles in a Teams meeting:

* **Organizer**: The organizer schedules a meeting, sets the meeting options, assigns meeting roles, and starts the meeting. Only users with a Microsoft 365 account and Teams license can be the organizer, and control attendee permissions. A meeting organizer can change the settings for a specific meeting. Organizers can make these changes on the **Meeting options** web page.

* **Presenter**: Presenters in a meeting have similar capabilities as the organizer, with the exception of removing an organizer from the session and modifying meeting options for the session.

* **Attendee**: An attendee is a user who is invited to attend the meeting. Attendees have limited capabilities during the meeting.

> [!NOTE]
>
> * Only an organizer or presenter can add, remove, or uninstall apps.

> > * The default participant settings are determined by an organization's IT administrator.

Reviewer 3: @v-sdhakshina  pls remove line 75 its duplicate in tips

* [**Group calls**](https://support.microsoft.com/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in group chat.
* [**Instant meetings**](https://support.microsoft.com/office/start-an-instant-meeting-in-teams-ff95e53f-8231-4739-87fa-00b9723f4ef5): Meetings initiated through **Meet now** button in Teams calendar.

> [!NOTE]
>
> Apps for instant meetings, one-on-one, and group calls are currently available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).

Learn more about [Teams meetings, expiration and policies](/MicrosoftTeams/meeting-expiration) and [Meetings, webinars, and live events](/microsoftteams/quick-start-meetings-live-events).

> Users can add apps to the meeting using the **+** option from their Teams meeting window.

Author: removed
